README.md

```diff
@@ -69,13 +71,13 @@ We welcome any form of contributions, code, documentation or even discussions. T
 
 **Communication channels**
 
-- We use [Slack](https://slack.com/) as our primary collaboration system, you can join us by accessing [this link](https://join.slack.com/t/yunikornworkspace/shared_invite/enQtNzAzMjY0OTI4MjYzLTBmMDdkYTAwNDMwNTE3NWVjZWE1OTczMWE4NDI2Yzg3MmEyZjUyYTZlMDE5M2U4ZjZhNmYyNGFmYjY4ZGYyMGE).
+- Mailing lists are:
+  - user@ user questions list
+  - dev@ development discussions
+  - issues@ JIRA issue feed
```

Comment:
Why not use complete mailing list addresses here? e.g user@yunikorn.incubator.apache.org

Reply:
oops, I thought I had fixed that new commit fixes all links